flutter/lib/benchmark/resource_manager.dart

```diff
@@ -136,6 +136,17 @@ class ResourceManager {
     await Directory(loadedResourcesDir).create();
   }
 
+  Future<void> createConfigurationFile() async {
+    final file = File('$applicationDirectory/$_configurationsFileName');
+    if (!await file.exists()) {
+      print('Create ' + file.path);
+      var config = <String, String>{
+        defaultBenchmarksConfiguration.name: defaultBenchmarksConfiguration.path
```

Comment:
If we update the app and default configuration URL changes, but we already have the `benchmarksConfigurations.json` file, the app will not use the new URL unless we clear app data or manually delete `benchmarksConfigurations.json`.
Is this the expected behavior?

Reply:
I remember there was a discussion how we should handle updated default configuration. There are cases:
1. The URL is changed but app is not changed.
2. The URL is not changed but the content of the config file is changed.

This would be best to handle in a separated issue.

Reply:
Could you open a new issue for this after we merge this PR?

Reply:
https://github.com/mlcommons/mobile_app_open/issues/184